<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite skipped="1" hostname="IN-SATHEESHNAI" name="regression.TC_AddNewResource" tests="7" failures="1" timestamp="04 Mar 2019 11:45:39 GMT" time="181.242" errors="0">
  <testcase name="loginasvendor" time="11.740" classname="regression.TC_AddNewResource"/>
  <testcase name="clickontask" time="5.885" classname="regression.TC_AddNewResource"/>
  <testcase name="clickonresource" time="6.463" classname="regression.TC_AddNewResource"/>
  <testcase name="addnewresource" time="62.471" classname="regression.TC_AddNewResource">
    <failure type="java.lang.AssertionError" message="Failed to Click on Add position button">
      <![CDATA[java.lang.AssertionError: Failed to Click on Add position button
at org.testng.Assert.fail(Assert.java:96)
at pages.ResourcesPage.clickaddresourcebtn(ResourcesPage.java:152)
at regression.TC_AddNewResource.addnewresource(TC_AddNewResource.java:55)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- addnewresource -->
  <testcase name="enterresourcedetails" time="87.471" classname="regression.TC_AddNewResource"/>
  <testcase name="submitresource" time="7.212" classname="regression.TC_AddNewResource"/>
  <testcase name="logout" classname="regression.TC_AddNewResource">
    <skipped/>
  </testcase> <!-- logout -->
</testsuite> <!-- regression.TC_AddNewResource -->
